coauthors.go

```diff
@@ -3,6 +3,7 @@ package main
 import (
 	"bufio"
 	"fmt"
+	"github.com/remotemobprogramming/mob/v3/say"
```

Comment:
I'am not sure if this is the way to go. But disclaimer I also didn't knew that before I just looked up if there is something like private. For me this looks a bit strange but maybe I do not know how it correctly works.

Reply:
I just read a bit about packages. This should be fine :)